.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
+
+install:
+  
+  - pip install -r requirements.pip
+
+services:
+  - postgresql
+
+script: python django_imager/manage.py py.test --no input
```

Comment:
This script it wrong. to run tests in django you do `python django_imager/manage.py test`